app/Enums/DatabaseDriver.php

```diff
@@ -0,0 +1,124 @@
+            self::Mysql => 'MySQL',
+            self::Postgresql => 'PostgreSQL',
+            self::Sqlite => 'SQLite',
+        };
+    }
+
+    public function getJDBCDriver(): string
+    {
+        return match($this) {
+            self::Mariadb, self::Mysql => 'mysql',
+            self::Postgresql => 'postgresql',
+            self::Sqlite => 'sqlite',
+        };
+    }
+
+    public function isRemote(): bool
+    {
+        return match($this) {
+            self::Mariadb, self::Mysql, self::Postgresql => true,
+            self::Sqlite => false,
+        };
+    }
+
+    public function getDefaultOption(string $key, bool $useEnv = false): mixed
+    {
+        $defaultValue = (match($this) {
+            self::Mariadb => [
+                'driver' => 'mariadb',
+                'url' => '',
+                'host' => '127.0.0.1',
+                'port' => 3306,
+                'database' => 'panel',
+                'username' => 'pelican',
+                'password' => '',
+                'unix_socket' => '',
+                'charset' => 'utf8mb4',
+                'collation' => 'utf8mb4_unicode_ci',
+                'prefix' => '',
+                'prefix_indexes' => true,
+                'strict' => false,
+                'engine' => null,
+                'options' => [],
+                'test_database' => 'mysql',
+            ],
+            self::Mysql => [
+                'driver' => 'mysql',
+                'url' => '',
+                'host' => '127.0.0.1',
+                'port' => 3306,
+                'database' => 'panel',
+                'username' => 'pelican',
+                'password' => '',
+                'unix_socket' => '',
+                'charset' => 'utf8mb4',
+                'collation' => 'utf8mb4_unicode_ci',
+                'prefix' => '',
+                'prefix_indexes' => true,
+                'strict' => false,
+                'engine' => null,
+                'options' => [],
+                'test_database' => 'mysql',
+            ],
+            self::Postgresql => [
+                'driver' => 'pgsql',
+                'url' => '',
+                'host' => '127.0.0.1',
+                'port' => 5432,
+                'database' => 'panel',
+                'username' => 'pelican',
+                'password' => '',
+                'charset' => 'UTF8',
+                'prefix' => '',
+                'prefix_indexes' => true,
+                'sslmode' => 'prefer',
+                'test_database' => 'postgres',
+            ],
+            self::Sqlite => [
+                'driver' => 'sqlite',
+                'url' => '',
+                'database' => 'database.sqlite',
+                'prefix' => '',
+                'foreign_key_constraints' => true,
+            ],
+        })[$key] ?? null;
+        return $useEnv ? config()->get(sprintf('database.connections.%s.%s', $this->value, $key), $defaultValue) : $defaultValue;
```

Comment:
Please use string interpolation as much as possible!

```suggestion
        return $useEnv ? config()->get("database.connections.{$this->value}.$key", $defaultValue) : $defaultValue;
```

Reply:
Wait I was thinking PHP still didn't have string interpolation... Whoops, I'll have to fix that everywhere.